sopel/modules/wikipedia.py

```diff
@@ -189,7 +189,11 @@ def say_snippet(bot, trigger, server, query, show_url=True):
         snippet = re.sub(r"\s+", " ", snippet)
     except KeyError:
         if show_url:
-            bot.reply("Error fetching snippet for \"{}\".".format(page_name))
+            msg = 'Error fetching snippet for "{}".'.format(page_name)
```

Comment:
I was going to replace some of these simpler `str.format()` calls with f-strings (which should be fine since Sopel is 3.7+ now?) but I notice that the library doesn't use them anywhere else.

```suggestion
            msg = f'Error fetching snippet for "{page_name}".'
```

Reply:
Clarified on IRC: this is a stylistic choice!